build.gradle

```diff
@@ -24,4 +24,7 @@ dependencies {
     compile('org.springframework.boot:spring-boot-actuator')
     compile('org.springframework.boot:spring-boot-starter-web')
     compile('com.nfl.graphql:gold:1.+')
+    compile('com.graphql-java:graphql-java:4.2')
+    compile('com.nfl.graphql:mediator:1.0.1')
```

Comment:
Gold should reference the graphql-java library.  These 3 includes don't seem necessary to me.

Reply:
They should be brought in transitively.